src/test/java/com/fasterxml/jackson/core/ErrorReportConfigurationMaxErrorTokenLengthTest.java

```diff
@@ -0,0 +1,132 @@
+package com.fasterxml.jackson.core;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Unit tests for class {@link ErrorReportConfiguration#getMaxErrorTokenLength()}.
+ */
+public class ErrorReportConfigurationMaxErrorTokenLengthTest extends BaseTest
```

Comment:
How about combine various tests related to `ErrorReportConfiguration`, instead of separate test classes? Or at least these two.